data/materials/scents.json

```
@@ -0,0 +1,179 @@
+	"scent": [
```

Thanks for the PR!

We need to wrap the whole thing inside `{}`

And I think we need to add a "description" as well?

```suggestion
{
		"description": "Scents taken from various perfume notes",
		"scent": [
```